.github/workflows/issue_stale_bot.yml

```diff
@@ -8,6 +8,7 @@ on:
 permissions:
   contents: read
   issues: write
+  pull-requests: write
```

Comment:
Why is pull request permission added, but not used?

Reply:
> Why is pull request permission added, but not used?

I couldn't find the exact failed run, but I saw once it tried to detect the PR status and complained it didn't have the permission - could even be an upstream problem.